packages/node/src/builders/package/package.impl.ts

```diff
@@ -209,6 +370,28 @@ function updatePackageJson(
   packageJson.typings = normalize(
     `./${options.relativeMainFileOutput}/${typingsFile}`
   );
+
+  // add any dependency to the dependencies section
+  packageJson.dependencies = packageJson.dependencies || {};
+  libDependencies.forEach(entry => {
+    if (!packageJson.dependencies[entry.scope]) {
```

Comment:
I think this check has to be extended:

- The depedency can be a dev dependency or a peer dependency.  I'd check if it appers in any of them, then we should not update anything.

Reply:
Same applies to the angular builder

Reply:
true! gonna fix that tomorrow 👍